### ISSUE TITLE
Why the "dnspython" requirement version is locked to a specific and obsolete one ?

### ISSUE THREAD

**Issue by maaaaz** [atenreiro/opensquat#84]
Hello there,

Everything is in the title :)
https://github.com/atenreiro/opensquat/blob/295d58a4f33723d5985cadbfb4984401e9ee84ed/requirements.txt#L14

Cheers!

**Comment by atenreiro** [comment]
Hello!

There was a bug with the latest version of dnspython available at that time, hence I fixed into a specific build to prevent the issue. 

I rever revisit the latest releases to confirm that the big as been fixed but thanks for pointing this out.

**Comment by maaaaz** [comment]
Hello there,

I dug, and indeed the current code use the [`query`](https://github.com/atenreiro/opensquat/blob/295d58a4f33723d5985cadbfb4984401e9ee84ed/opensquat/dns_resolvers.py#L45) deprecated function:
```
>>> import dns.resolver
>>> my_resolver = dns.resolver.Resolver(configure=False)
>>> my_resolver.nameservers = ['9.9.9.9']
>>> my_resolver.search = []
>>> my_resolver.query("google.fr", "A")
<stdin>:1: DeprecationWarning: please use dns.resolver.Resolver.resolve() instead
<dns.resolver.Answer object at 0x7f381af2af50>
```

I fixed by using "resolve" function instead of "query".
=> https://github.com/atenreiro/opensquat/pull/85

Cheers !

**Comment by atenreiro** [comment]
Fixed in the pull #85
